cli/testdata/coder_users_list.golden

```diff
@@ -0,0 +1,3 @@
+USERNAME   EMAIL                CREATED AT            STATUS   
+testuser   testuser@coder.com   [timestamp]  active   
```

Comment:
Can we replace it with the same length string? Worried someone might think this is a bug. 😄

Reply:
It then looks like this:
```
USERNAME   EMAIL                CREATED AT            STATUS   
testuser   testuser@coder.com   [timestamp         ]  active   
testuser2  testuser2@coder.com  [timestamp         ]  dormant  

```

Reply:
And in json
```json
    "created_at": "[timestamp                ]",
    "last_seen_at": "[timestamp                ]",
```